test/integration/organization-space-membership-integration.js

```diff
@@ -0,0 +1,22 @@
+export default function organizationSpaceMembershipTests (t, organization) {
+  t.test('Gets organizationSpaceMemberships', (t) => {
+    t.plan(2)
+    return organization.getOrganizationSpaceMemberships()
+      .then((response) => {
+        t.ok(response.sys, 'sys')
+        t.ok(response.items, 'fields')
```

Comment:
Should you check if `items` contain space memberships?

Reply:
@guilebarbosa I am checking it with the `getOrganizationSpaceMembership()` test. But maybe I would adjust it to check it more explicitly like: 
```
t.equal(response.sys.type, 'SpaceMembership', 'type')
```
WDYT?

Reply:
Hmm, actually the type here would be `Array`.

Reply:
@guilebarbosa I was mentioning the test which I would add into the `getOrganizationSpaceMembership`. I am not sure if I should test it in here...

Reply:
Oh, sorry! 🤦🏻‍♂️
Yeah, that would be it.